sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -154,7 +154,8 @@ describe("SmsClient [Playback/Live]", async () => {
       );
       assert.fail("Should have thrown an error");
     } catch (e) {
-      assert.equal(e.statusCode, 401);
+      // TODO: re-enable this when service change is made
```

Comment:
if we want to be fancy we can create a github issue or ADO task and put the link here